src/pages/cart/Cart.css

```diff
@@ -276,6 +276,29 @@
   font-size: 1rem;
 }
 
+.card__add-cart-btn--in-cart {
+  background-color: var(--text2);
+  color: var(--light-clr);
+}
+
+.card__add-cart-btn:disabled {
+  cursor: not-allowed;
+}
+
+.card__update-quantity--add, 
+.card__update-quantity--remove {
+  cursor: pointer;
+}
+
+.card__update-quantity--add:disabled, 
+.card__update-quantity--remove:disabled {
+  cursor: not-allowed;
+}
+
+.cart-icon-quantity {
+  font-size: .8rem !important;
```

Comment:
Better to avoid using !important, other than these everything looks good

Reply:
Using this I'm overriding material icon default styling. It is not applying without `imortant`.

Reply:
try increasing specificity by adding another class for icon styling. i believe it will work